docs/developer/centrifuge-chain/evm/index.md

```diff
@@ -0,0 +1,56 @@
+---
+id: evm
+order: 4
```

Comment:
Seeing the preview now, I would change the order to
- Overview
- EVM compatibility
- Networks
- Codebase
- Contributing